File: backend/src/main/java/capstone/rt04/retailbackend/request/staff/StaffCreateRequest.java
package capstone.rt04.retailbackend.request.staff;

import capstone.rt04.retailbackend.entities.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class StaffCreateRequest {

    private Staff staff;
    private Address staffAddress;
    private Long roleId;
    private Long departmentId;

}

File: frontend/staff-web-app/src/models/staff/StaffCreateRequest.js
export default class StaffCreateRequest {
  staff;
  staffAddress;
  roleId;
  departmentId;
  storeId;

  constructor(staff, roleId, departmentId, staffAddress, storeId) {
    this.staff = staff;
    this.staffAddress = staffAddress;
    this.roleId = roleId;
    this.departmentId = departmentId;
    this.storeId = storeId;
  }
}
